File: MojPrijevozFront/moj_prijevoz_admin/lib/resources/responses/city/city_response.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:moj_prijevoz_admin/resources/responses/city/all_cities_response.dart';

part 'city_response.g.dart';

@JsonSerializable()
class CityResponse extends AllCitiesResponse {
  CityResponse({
    required super.id,
    required super.name,
    required super.lat,
    required super.long,
    required super.createdAt,
    required super.updatedAt,
  });

  factory CityResponse.fromJson(Map<String, dynamic> json) =>
      _$CityResponseFromJson(json);

  @override
  Map<String, dynamic> toJson() => _$CityResponseToJson(this);
}

File: MojPrijevoz/MojPrijevoz.Model/Responses/City/CityResponse.cs
namespace MojPrijevoz.Model.Responses.City;

public class CityResponse {
    public int Id { get; set; }
    public string Name { get; set; } = null!;
    public string Long { get; set; } = null!;
    public string Lat { get; set; } = null!;
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
}